client-metrics/src/main/scala/org/http4s/client/metrics/core/MetricsOps.scala

```diff
@@ -0,0 +1,75 @@
+package org.http4s.client.metrics.core
```

Comment:
If we're going the route of making this common, this needs to move out of client.

Reply:
Yes, that's true. Where should we put it? in core?

Reply:
Yeah, how about in http4s-core, in `org.http4s.metrics`?  It doesn't pull in a new dependency, so I don't see it going into its own module.

Reply:
sure. I'll move it there